Issue in parsl.log

I tried to run a simple script given in the parsl documentation 

```
import parsl
from parsl import *
import time

workers = ThreadPoolExecutor(max_workers=4)
dfk = DataFlowKernel(executors=[workers])
print(1)
@App('python', dfk)
def hello ():
    import time
    time.sleep(5)
    return 'Hello World!'
print(2)
app_future = hello()
print ('Done: %s' % app_future.done())
print ('Result: %s' % app_future.result())
print ('Done: %s' % app_future.done())
```
However, in the parsl.log shows this issue

2018-06-07 21:45:37 parsl.utils:24 [ERROR]  Unable to determine code state
Traceback (most recent call last):
  File "/homes/vvikraman/anaconda3/lib/python3.6/site-packages/parsl/utils.py", line 19, in get_version
    head = subprocess.check_output(cmd, env=env).strip().decode('utf-8')
  File "/homes/vvikraman/anaconda3/lib/python3.6/subprocess.py", line 336, in check_output
    **kwargs).stdout
  File "/homes/vvikraman/anaconda3/lib/python3.6/subprocess.py", line 418, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['git', 'rev-parse', '--short', 'HEAD']' returned non-zero exit status 128.

--- a/parsl/utils.py
+++ b/parsl/utils.py
@@ -25,7 +25,7 @@ def get_version():
         status = 'dirty' if diff else 'clean'
         version = '{v}-{head}-{status}'.format(v=VERSION, head=head, status=status)
     except Exception as e:
-        logger.exception("Unable to determine code state")
+        pass
 
     return version
 

